lib/ramfs/ramfs.h

```diff
@@ -36,29 +36,65 @@
 #include <vfscore/prex.h>
 #include <stdbool.h>
 
-/*
- * File/directory node for RAMFS
+/**
+ * struct ramfs_node - A file/directory node for RamFS.
```

Comment:
We should be consistent in using `RamFS` or `RAMFS` or `ramfs` throughout the file.

Reply:
How do you recommend we change this? `ramfs` is used when referring to the name of the structure `ramfs_node`, `RAMFS` is used for the macro `RAMFS_NODE`, and in `ifndef`, `define`, and `endif`. Should we change `RamFS` to one of these?